### DISCUSSION TITLE
Variable from Webhook request by Flow

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15454] [Archive]
Hi,

- by Flow I have a Webhook request with "forecast_id" (see screenshot of Webhook Payload)
![variable](https://user-images.githubusercontent.com/54798086/188886131-805d0250-925c-4796-b16f-10573f403ef1.PNG)

- in the next step of Flow, I need to Update Data (see screenshot) 
![variable2](https://user-images.githubusercontent.com/54798086/188885406-eaefffb2-bbfc-4827-9c86-70929a671f8f.PNG)

What a name of variable for "forecast_id" from Webhook Request must I use to update a field "forecast_id"? At the moment i have set there a manual number "1001000666", but instead of that number must be variable...

Best regards,
Tony

**Comment by rynskiy** [comment]
Is anyone here? :)

**Reply by tmdoit** [reply]
Hi @rynskiy,
you can use this: `"{{<operation_key>.data.forecast_id}}"` where `operation_key` is the webhook key (you can get it from webhook step edit screen).

**Reply by rynskiy** [reply]
Thanks, @tmdoit! It's works perfect!

**Comment by br41nslug** [comment]
https://docs.directus.io/configuration/flows.html#the-flow-object
